packages/shylu/shylu_dd/frosch/src/SchwarzPreconditioners/FROSch_OneLevelPreconditioner_def.hpp

```diff
@@ -141,6 +163,15 @@ namespace FROSch {
         return "One-Level Preconditioner";
     }
     
+    template <class SC,class LO,class GO,class NO>
+    int OneLevelPreconditioner<SC,LO,GO,NO>::resetMatrix(CrsMatrixPtr &k)
+    {
+        K_ = k;
+        OverlappingOperator_->resetMatrix(K_);
+        if (UseMultiplicative_) MultiplicativeOperator_->resetMatrix(K_);
+        return 0;
```

Comment:
One more question, after I gave the thumbsup : What happens in sumOperator ? No need to reset ?

Reply:
@srajama1 well spotted, and indeed not needed. Actually, I deleted the reset of this MultiplicativeOperator in my recent commit, as it only makes sense for a two level preconditioner. In case of a two level preconditioner the reset is only needed for a MultiplicativeOperator, since the problem matrix K_ is applied inside the preconditioner. This is not the case for the SumOpertor, therefore K_ is not needed there.

Reply:
Thanks ! It is somewhat confusing to me. As long as you and @searhein agree, I am fine with it.